internal/venice-test-common/src/integrationtest/java/com/linkedin/venice/controller/TestHAASController.java

```diff
@@ -125,6 +125,7 @@ public void testTransitionToHAASControllerAsControllerClusterLeader() {
               () -> assertEquals(
                   assertCommand(controllerClient.queryJobStatus(topicName)).getStatus(),
                   ExecutionStatus.COMPLETED.toString())));
+      System.out.println("End of the test");
```

Comment:
Same here, but since this it test, probably fine to keep it (probably for the sake of debugging flakiness..)

Reply:
Thanks for catching it... I thought I had removed it, but forgot. Will clear it in the next commit.